internal/commands/run.go

```diff
@@ -133,20 +134,33 @@ func run(cmd *cobra.Command, args []string) error {
 
 	var executorOpts []executor.Option
 
+	tryToDetectOutput := output == "auto"
+	if tryToDetectOutput && envVariableIsTrue("TRAVIS") {
+		output = "travis"
```

Comment:
Perhaps we should make these strings constants, since they're used multiple times?

In this case they can also be re-used in the flag description.

Reply:
Thought of it but Go doesn't have enums. :-(